js/renovation/ui/scroll_view/scroll_view.tsx

```diff
@@ -44,41 +38,16 @@ export const viewFunction = (viewModel: ScrollView): JSX.Element => {
       {...props}
       // eslint-disable-next-line react/jsx-props-no-spreading
       {...restAttributes}
+      pulledDownText={pulledDownText}
+      pullingDownText={pullingDownText}
+      refreshingText={refreshingText}
       forceGeneratePockets
       needScrollViewContentWrapper
     />
   );
 };
-@ComponentBindings()
-export class ScrollViewProps extends ScrollableProps {
-  @OneWay() refreshStrategy: RefreshStrategy = 'pullDown';
-
-  @OneWay() pullingDownText = messageLocalization.format('dxScrollView-pullingDownText');
-
-  @OneWay() pulledDownText = messageLocalization.format('dxScrollView-pulledDownText');
-
-  @OneWay() refreshingText = messageLocalization.format('dxScrollView-refreshingText');
-
-  @OneWay() reachBottomText = messageLocalization.format('dxScrollView-reachBottomText');
-}
-
-export type ScrollViewPropsType = ScrollViewProps & Pick<BaseWidgetProps, 'rtlEnabled' | 'disabled' | 'width' | 'height'>;
-
-export const defaultOptionRules = createDefaultOptionRules<ScrollViewProps>([{
-  device: (): boolean => devices.real().platform === 'android',
-  options: { refreshStrategy: 'swipeDown' },
-}, {
-  // eslint-disable-next-line import/no-named-as-default-member
-  device: (): boolean => Themes.isMaterial(Themes.current()),
-  options: {
-    pullingDownText: '',
-    pulledDownText: '',
-    refreshingText: '',
-    reachBottomText: '',
```

Comment:
reachBottomText is not rewritten

Reply:
Rewritten in https://github.com/DevExpress/DevExtreme/pull/15914/commits/29a91afaca19b72b35c608c1b3171332a569590d